CHANGELOG.md

```diff
@@ -16,10 +16,13 @@ All user visible changes to this project will be documented in this file. This p
 - UI:
     - Profile page:
         - Avatar cropping. ([#1139], [#1130], [#530])
+    - Chat page:
+        - Image and file attachments pasting from pasteboard. ([#1141])
```

Comment:
Why is 7f1cb16cc93e7f369c22c2dcb7b2d2c48d672a5f displayed in the diff with the main branch?

Reply:
The repository was updated after creating the pull request. Hence, it is displayed in the diff with the main branch. But, since, there is no change it can be merged with no conflicts.

Reply:
I did merge but after opening this pull request. I should have done it before. Can we please proceed this time?

Reply:
@pratham-sarankar, did you check the documentation of `git rebase` or `git merge`? It doesn't matter when you've opened the PR, I'm asking you to merge the latest main branch onto yours - why can't you accomplish that, is there a concrete problem when running the merge?

Reply:
@SleepySquash I did `git rebase` and `git --force push`. Now it seems to be resolved.